pkg/covermerger/provider_monorepo.go

```diff
@@ -79,7 +98,8 @@ func (mr *monoRepo) addRepoCommit(rbc RepoBranchCommit) {
 func MakeMonoRepo(workdir string) FileVersProvider {
 	rbcPath := workdir + "/repos/linux_kernels"
```

Comment:
Use filepath.Join.
Direct `+` may break if `/` is already present in workdir, and won't work on e.g. Windows.

Reply:
Let's do it in #5230.